roles/network_plugin/calico/tasks/check.yml

```diff
@@ -43,10 +43,16 @@
   run_once: True
   delegate_to: "{{ groups['kube_control_plane'][0] }}"
 
-- name: Check if calico exists
+- name: Check if calicoctl.sh exists
   stat:
     path: "{{ bin_dir }}/calicoctl.sh"
-  register: calico_exists
+  register: calicoctl_sh_exists
+  run_once: True
+  delegate_to: "{{ groups['kube_control_plane'][0] }}"
+
+- name: Check if calico crd clusterinformations exists
```

Comment:
Can the above two tasks be done in one task? Essentially, they do only the same thing: `check if calico exists`

Reply:
Due to ansible limition, one task just do one thing.